server/migrations/20170321114511-update-usermessageevents.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.addColumn('UserMessageEvents', 'fromUserId', {

```

Comment:
Instead, we need a userId that identifies the *viewer* of the message.  This is usually the toUserId of the message, but not necessarily so.